src/operations/write.ts

```diff
@@ -65,8 +65,9 @@ export const write = (
   if (operation.operation === 'query') {
     writeSelection(ctx, 'Query', select, data);
   } else {
-    const isMutation = operation.operation === 'mutation';
-    writeRoot(ctx, isMutation, select, data);
+    const typename =
```

Comment:
Maybe we should reintroduce the helper that converts query --> Query and so on

Reply:
probably

Reply:
done